addons/olap_tutorial/models/report_tutorial.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime, timedelta
+
+from odoo import models, fields, api, _
+
+
+class RTBRecord(models.Model):
+    _name = "olap_tutorial.record.receipt_to_bin"
+    _inherit = "olap.report.record"
+    _description = """Record the number of items received"""
+
+    picking_id = fields.Many2one(
+        'stock.picking', 'Picking', required=True, index=True)
+    po_number = fields.Char('PO Number')
+    scheduled_date = fields.Datetime('Scheduled Date')
+    num_lines = fields.Integer('Line Count')
+
+
+class RTBModel(models.AbstractModel):
+    _name = "olap_tutorial.model.receipt_to_bin"
+    _inherit = "olap.report.model"
+
+    _record_model = RTBRecord._name
+    _reported_model = 'stock.picking'
+
+    @api.model
+    def get_candidate_records(self):
+        """Get all pickings updated in the last 24 hours"""
+        goods_in_type = self.env.ref('stock.picking_type_in')
+        max_age = datetime.now() - timedelta(hours=24)
```

Comment:
couldn't you overwrite _default_from_date and use that? That way the max_age is always coming from one place